src/components/views/FigureDropdown.tsx

```diff
@@ -63,6 +65,7 @@ export const FigureElementOptions: React.FC<FigureElementOptionsProps> = ({
     .filter((f) => isImageFile(f.name))
 
   const others = files.others.filter((f) => isImageFile(f.name))
+  const showAddFile = onAddNewFile ? onAddNewFile() : true
 
   return (
     <FilesDropdownWrapper onClick={toggleOpen} ref={wrapperRef}>
```

Comment:
add `hasUploadedImage` to `FilesButton` disabled prop

Reply:
done